src/plugins/usage_collection/server/config.ts

```diff
@@ -11,6 +11,11 @@ import { PluginConfigDescriptor } from 'src/core/server';
 import { DEFAULT_MAXIMUM_WAIT_TIME_FOR_ALL_COLLECTORS_IN_S } from '../common/constants';
 
 export const configSchema = schema.object({
+  usageCounters: schema.object({
+    enabled: schema.boolean({ defaultValue: true }),
+    retryCount: schema.number({ defaultValue: 1 }),
+    bufferDurationMs: schema.duration({ defaultValue: '5s' }),
```

Comment:
Thanks for switching to `duration`. But now, I think we'll need to rename from `bufferDurationMs` to `bufferDuration` 😅

Reply:
@pgayvallet haha! true